src/models/searchservice/response/directanswer/BaseFieldValueDirectAnswer.ts

```diff
@@ -1,14 +1,15 @@
 import { DirectAnswer } from './DirectAnswer';
 import { DirectAnswerType } from './DirectAnswerType';
 import { BuiltInFieldType } from './BuiltInFieldType';
-import { Result } from './Result';
+import { Result } from '../Result';
+import { EnumOrLiteral } from '../../../utils/EnumOrLiteral';
 
 /**
  * A direct answer where the answer came from a field from the knowledge graph.
  *
  * @public
  */
-export interface FieldValueDirectAnswer<T = unknown> extends DirectAnswer<T> {
+export interface BaseFieldValueDirectAnswer<T = unknown> extends DirectAnswer<T> {
```

Comment:
What do you think about putting FieldValueDirectAnswer and BaseFieldValueDirectAnswer into the FieldValueDirectAnswer file?

Reply:
I really like each file doing "one thing" but it's also nice for stuff to be more consolidated if we have like 10-20 line files

Reply:
sure thing

Reply:
hmm I kinda like them separate, but if you feel strongly, we can group them

Reply:
sure separate is fine